core/ecschema-editing/src/Editing/Editor.ts

```diff
@@ -201,5 +201,32 @@ export class SchemaContextEditor {
 
     return schema;
   }
+
+  public async lookupSchemaItem<T extends SchemaItem>(schemaOrKey: Schema | SchemaKey, schemaItemKey: SchemaItemKey, schemaItemType: SchemaItemType): Promise<T>{
```

Comment:
I wonder if we should mark it internal to make clear this function is not supposed to be called by anything other than the actual editor method.

Reply:
Done